util/bidderutil/bidderutil.go

```diff
@@ -0,0 +1,27 @@
+package bidderutil
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+)
+
+func CheckResponseStatusCodeForErrors(response *adapters.ResponseData) error {
+	if response.StatusCode == http.StatusBadRequest {
+		return &errortypes.BadInput{
+			Message: fmt.Sprintf("Unexpected status code: %d. Run with request.debug = 1 for more info", response.StatusCode),
+		}
+	}
+
+	if response.StatusCode != http.StatusOK {
+		return fmt.Errorf("Unexpected status code: %d. Run with request.debug = 1 for more info", response.StatusCode)
+	}
+
+	return nil
+}
+
+func IsResponseCodeStatusNoContent(response *adapters.ResponseData) bool {
```

Comment:
Generally we say `http status code`. From this perspective, how about renaming `IsResponseCodeStatusNoContent` to `IsResponseStatusCodeNoContent`  or `hasNoContentStatusCode` ?

Reply:
[87d05e0](https://github.com/prebid/prebid-server/pull/2638/commits/87d05e0af3b5db96bcbc7e269be6d79a3c99a522)